packages/connect/src/utils/debug.ts

```diff
@@ -109,3 +152,12 @@ export const getLog = () => {
     logs.sort((a, b) => a.timestamp - b.timestamp);
     return logs;
 };
+
+export const initSharedLogger = (workerSrc: string) => {
+    const worker = new SharedWorker(workerSrc);
+    worker.port.start();
+    const logWriterFactory = (): LogWriter => ({
+        add: (message: LogMessage) => worker.port.postMessage({ type: 'add-log', data: message }),
+    });
+    setLogWriter(logWriterFactory);
```

Comment:
hmm I went to merge this already but this caught my eye

in packages/connect, there should be only code that runs in both browser && node. SharedWorker is already some superset. 

I suggest moving this to 
`packages/connect-web/src/utils/debug` 

or maybe better to 

`packages/connect-popup/src/utils/debug`

Reply:
we should also reconfigure typescript not to allow such thing in this packag, one day...

Reply:
Yes, that makes sense since now it is use only in `connect-popup`. :+1: 
Done :heavy_check_mark: